specification/vmware/resource-manager/readme.go.md

```diff
@@ -5,17 +5,27 @@ These settings apply only when `--go` is specified on the command line.
 ``` yaml $(go)
 go:
   license-header: MICROSOFT_APACHE_NO_VERSION
-  namespace: vmware
+  namespace: avs
   clear-output-folder: true
 ```
 
 ### Go multi-api
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2020-03-20
   - tag: package-2019-08-09-preview
 ```
 
+### Tag: package-2020-03-20 and go
+
+These settings apply only when `--tag=package-2020-03-20 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-03-20' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-03-20/$(namespace)
```

Comment:
I think 2020-03-20 is a stable version,  and the output-folder should be `$(go-sdk-folder)/services/$(namespace)/mgmt/2020-03-20/$(namespace)`. It should not be in preview folder

Reply:
Thanks. Removed in b6695fe6da6a1eecc2fbfe3df8cf3eb1d98f424e.